quill-async-postgres/src/test/scala/io/getquill/context/async/postgres/ArrayExtraAsyncEncodingSpec.scala

```diff
@@ -0,0 +1,56 @@
+package io.getquill.context.async.postgres
+
+import java.time.{ ZonedDateTime }
+
+import io.getquill.context.sql.encoding.ArrayEncodingExtraSpec
+import org.joda.time.{ DateTime => JodaDateTime }
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ArrayExtraAsyncEncodingSpec extends ArrayEncodingExtraSpec {
```

Comment:
there's no need of separate suite, just add you unit test to existing ArrayAsyncEncodingSpec